frameworks/digital-outcomes-and-specialists-2/questions/briefs/status.yml

```diff
@@ -1,9 +0,0 @@
-name: Status
-id: status
-question: status
-type: radios
-options:
-  -
-    label: open
-    value: live
```

Comment:
As discussed yesterday, we'll need to keep this in for the moment otherwise the Buyer FE won't render the filter form properly.

Reply:
Is this intermediate version even intended to be pulled into the buyer FE?

Reply:
Maybe not by us, but someone else might! And it would certainly be a breaking change if we left this change in.

Reply:
I don't think this is a breaking change - if the buyer frontend is updated to point to the new frameworks version, then it will also be using the new field name - which we will have indexed.

Reply:
(It is breaking only insofaras the URL's will be changing, but we don't guarantee that as an API and I'm considering that to be acceptable, although contrary views welcome.)

Reply:
*Massive shrug*

Reply:
After chatting I'm convinced it'll all be OK 😸